packages/ui/src/modal-building-code-content/ModalBuildingCodeContent.tsx

```diff
@@ -0,0 +1,227 @@
+"use client";
+
+import {
+  ArticleType,
+  SubClauseType,
+  PartType,
+  SentenceType,
+  SectionType,
+  SubsectionType,
+} from "@repo/data/useGlossaryData";
+import { Heading } from "react-aria-components";
+import {
+  parseStringToComponents,
+  stripReferencePrefix,
+} from "web/utils/string";
+import Image from "../image/Image";
+import "./ModalBuildingCodeContent.css";
+
+interface BuildingCodeContentProps {
+  modalData: PartType[];
+  highlightedSection: string | null;
+  sectionRefs: React.MutableRefObject<{ [key: string]: HTMLElement | null }>;
+  setFocusSection: (section: string) => void;
+}
+
+const BuildingCodeContent: React.FC<BuildingCodeContentProps> = ({
+  modalData,
+  highlightedSection,
+  sectionRefs,
+  setFocusSection,
+}) => {
+  const renderSubClauses = (subClauses: string[]) => {
+    return (
+      <ol type="i" className="ui-ModalBuildingCodeContent--List">
+        {subClauses.map((subClause, index) => (
+          <li key={index}>
+            <span>{parseStringToComponents(subClause, setFocusSection)}</span>
+          </li>
+        ))}
+      </ol>
+    );
+  };
+
+  const renderClauses = (clauses: SubClauseType[]) => {
+    return (
+      <ol type="a" className="ui-ModalBuildingCodeContent--List">
+        {clauses.map((data, index) => (
+          <div
```

Comment:
Same here as with the `<section>` elements. The `<div>`s should be inside of the `<li>`s semantically.

Reply:
Some of this will be addressed in the immediately following PR. Also, I do need some sort of html wrapper for the list items or else the ordered list will not have numbers. If I do the reference on the list item itself then the numbers go away. 

If you want to play around with it during the polishing phase I think that would be fine.